analysis/cohort_joiner.py

```diff
@@ -31,8 +31,22 @@ def match_paths(pattern):
 
 def parse_args():
     parser = argparse.ArgumentParser()
-    parser.add_argument("--lhs", dest="lhs_paths", required=True, type=match_paths)
-    parser.add_argument("--rhs", dest="rhs_paths", required=True, type=match_paths)
+    parser.add_argument(
+        "--lhs",
+        dest="lhs_paths",
+        required=True,
+        type=match_paths,
+        metavar="LHS_PATTERN",
+        help="Glob pattern for matching one or more dataframes that will form the left-hand side of the join",
```

Comment:
Not suggesting any changes here, i'm sure this will be covered in the docs, but it would be nice to explain what the left hand side of a left join means clearly for those unfamiliar